README.md

```diff
@@ -141,6 +142,69 @@ haskell_library(
   </tbody>
 </table>
 
+### haskell_library
```

Comment:
Look at rendered version of this README. We now have two `haskell_library` sections.